plugins/greynoise/help.md

```diff
@@ -273,158 +401,486 @@ Example input:
 
 ##### Output
 
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|classification|string|False|GreyNoise Classification|
-|ip|string|False|Value that was Queried|
-|last_seen|string|False|Last Seen By GreyNoise|
-|link|string|False|Link to GreyNoise Visualizer for IP Details|
-|message|string|False|GreyNoise Community API Status Message|
-|name|string|False|GreyNoise Actor or Service Name Associated with IP|
-|noise|boolean|False|Defines if IP is Internet Noise|
-|riot|boolean|False|Defines if IP is part of GreyNoise RIOT dataset|
-
+|Name|Type|Required|Description|Example|
+| :--- | :--- | :--- | :--- | :--- |
+|ip|ip_sim|False|IP Similarity Metadata|None|
+|similar_ips|[]similar_ip|False|Similar IPs|None|
+|total|integer|False|Total Number of Similar IPs returned|5|
+  
 Example output:
 
 ```
 {
-  "classification": "benign",
-  "ip": "1.1.1.1",
-  "last_seen": "2021-04-20T00:00:00+00:00",
-  "link": "https://viz.greynoise.io/riot/1.1.1.1",
-  "message": "Success",
-  "name": "Cloudflare Public DNS",
-  "noise": false,
-  "riot": true
+  "ip": {
+    "ASN": {},
+    "Actor": "",
+    "City": {},
+    "Classification": {},
+    "Country": {},
+    "Country Code": {},
+    "First Seen": {},
+    "IP Address": {},
+    "Last Seen": {},
+    "Organization": {}
+  },
+  "similar_ips": [
+    {
+      "ASN": {},
+      "Actor": "",
+      "City": {},
+      "Classification": {},
+      "Country": {},
+      "Country Code": {},
+      "Features Matched": [
+        {}
+      ],
+      "First Seen": {},
+      "IP Address": {},
+      "Last Seen": {},
+      "Organization": {},
+      "Similarity Score": 0.0
```

Comment:
The same here. That example output should be added to `plugin.spec.yaml` file. After that please refresh the plugin.